app/controllers/supervisors_controller.rb

```diff
@@ -12,7 +12,10 @@ class SupervisorsController < ApplicationController
 
   def index
     authorize Supervisor
-    @supervisors = policy_scope(current_organization.supervisors)
+    @supervisors = policy_scope(current_organization.supervisors).includes(
```

Comment:
Do you know what this policy scope is adding to the query? could we move the includes? or is it reused elsewhere where the preloading isn't needed?

Reply:
good call. if i understood you correctly, i moved the `.includes` to the specific block in the .html.erb where the `volunteers` and other relations are accessed

Reply:
i'm not sure about the policy_scope. i did not see other uses of `@supervisors` atm. is there a place you think itd be better to move it to?

Reply:
I'm not making a suggestion I was just posing another angle that you might consider, since it might be less code duplication.

You could move the includes bits to https://github.com/rubyforgood/casa/blob/main/app/policies/supervisor_policy.rb and have something like https://github.com/rubyforgood/casa/blob/4d0d12f7eb4d40f6b823d3d8d42300c736be233c/app/policies/hearing_type_policy.rb#L2 in there.